src/js/api/order.js

```diff
@@ -549,6 +547,155 @@ ripe.Ripe.prototype.logOrderP = function(number, options) {
     });
 };
 
+/**
+ * Returns the chat of an order.

```

Comment:
Perhaps add more information like:
```suggestion
 * Returns the general chat of an order, not related to any state.
```